test/functional/test_k8s_v1beta1_stateful_set.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
+
+import copy
+
+import pytest
+
+from openshift.helper.exceptions import KubernetesException
+
+version_mismatch = pytest.mark.skipif(
+    float(pytest.config.getoption('--openshift-version')) < 1.5,
+    reason='This API is not supported in openshift versions < 1.5'
+)
```

Comment:
Instead of defining `version_mismatch` here, it might be better to move this to test/functional/conftest.py and rename it to `version_gte_1_5`. Then it would be available for other tests as well.